doc/modules/mod_stream_management.md

```diff
@@ -1,21 +1,101 @@
 ### Module Description
-Enables [XEP-0198: Stream Management](http://xmpp.org/extensions/xep-0198.html). 
-It is implemented mostly in `ejabberd_c2s`. 
-This module is just a "starter", to supply the configuration values to new client connections.
-It also provides a basic session table API and adds a new stream feature.
+
+Enables [XEP-0198: Stream Management](http://xmpp.org/extensions/xep-0198.html).
```

Comment:
Thanks for the detailed documentation @NelsonVides! I have one suggestion related to content organization. It seams to me that the most important goal of this documentation is to provide information what this module do, how to configure it and what are all the possible configuration options. In my opinion, it is more efficient for the reader if this kind of information is at the top. Technical details could be moved below. What do you think?

Reply:
Cool, that's a nice idea 👍